### PR TITLE
Initial Remote Content Docs

### PR DESCRIPTION
**Pull request by AMZN-alexpete**
## Change summary

Adds information on how to use, create and manage remote repositories.

### Submission Checklist:

*  [x] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [x] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [x] **Consistency** - Does the content consistently follow the [Style Guide](https://o3de.org/docs/contributing/to-docs/style-guide/quick-reference)?
* [x] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

File: content/docs/contributing/to-official-remote-repository/_index.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+linkTitle: To the Official O3DE Remote Repository 
+title: Contributing to the official O3DE remote repository
+description: O3DE maintains an official remote repository for O3DE owned content.
+weight: 500
+toc: true
+---
+
+O3DE maintains an official remote repository for O3DE owned content and the `repo.json` for the repository is managed at https://github.com/o3de/canonical.o3de.org and any changes are made using GitHub pull requests.
```
Comment (spham-amzn) [inline content/docs/contributing/to-official-remote-repository/_index.md:9]: Run on sentence

```suggestion
O3DE maintains an official remote repository for O3DE owned content. The `repo.json` for the repository is managed at https://github.com/o3de/canonical.o3de.org and any changes are made using GitHub pull requests.  

```

File: content/docs/user-guide/_index.md [hunk 2]
```diff
@@ -27,6 +27,7 @@ Read about [the features provided by Open 3D Engine](/docs/welcome-guide/feature
 | [Packaging](packaging/) | Learn how to package your O3DE-based product for release. |
 | [Programming](programming/) |  Learn about core engine programming concepts in O3DE. |
 | [Project Configuration](project-config/) | Learn how to configure and customize your O3DE project. |
+| [Remote Content](remote-content/) | Learn about Remote Content, a built-in way to share and download  Gems, Projects and Templates. |
```
Comment (spham-amzn) [inline content/docs/user-guide/_index.md:30]:
```suggestion
| [Remote Content](remote-content/) | Learn about Remote Content, a built-in way to share and download Gems, Projects and Templates. |
```

File: content/docs/user-guide/remote-content/remote-repository-overview.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        assetprocessor_settings.setreg
+    CMakeLists.txt
+    gem.json
+    preview.png
+    repo.json
+```
+
+#### Multiple Gems, Projects or Templates remote repository
+
+If you intend to have multiple Gems, Projects or Templates in your remote repository there are two approaches.
```
Comment (spham-amzn) [inline content/docs/user-guide/remote-content/remote-repository-overview.md:37]:
```suggestion
If you intend to have multiple Gems, Projects or Templates in your remote repository there are two approaches:

```

File: content/docs/user-guide/remote-content/update-a-remote-repository.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+We recommend that you use the Project Manager and O3DE CLI to test your remote repository before you upload the changes to GitHub, a web server or other source control.
+
+### Test with Project Manager
+
+You can add local remote repositories in the Project Manager and verify the contents appear as expected and downloads succeed.
+1. Add your local remote repository in Project Manager on the **Remote Sources** page on the **Engine** tab, by pressing the **Add Repository** button and then use the browse button to select the folder containing your `repo.json` file and then press the **Add** button.
+1. Select the remote repository in the list and verify the expected contents are listed in the right pane.
+1. Verify any added gems appear correctly in the **Gem Catalog**.
+1. Verify any added projects appear correctly on the **Projects** page.
+1. Verify any added templates appear correctly on the **Create a New Project** page.
```
Comment (spham-amzn) [inline content/docs/user-guide/remote-content/update-a-remote-repository.md:87]: Capitalize Gems, Projects, Templates for consistency
```suggestion
1. Verify any added Gems appear correctly in the **Gem Catalog**.
1. Verify any added Projects appear correctly on the **Projects** page.
1. Verify any added Templates appear correctly on the **Create a New Project** page.
```

File: content/docs/user-guide/remote-content/update-a-remote-repository.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+
+### Test with the O3DE CLI 
+
+You can add local remote repositories with the O3DE CLI and verify downloads succeed.
```
Comment (spham-amzn) [inline content/docs/user-guide/remote-content/update-a-remote-repository.md:91]: isn't `local` and `remote` contradictory?

File: content/docs/user-guide/remote-content/use-a-remote-repository.md [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+weight: 100
+toc: true
+---
+
+You can use the [Project Manager](#using-project-manager-to-access-o3de-remote-repositories#using-project-manager-to-access-o3de-remote-repositories) and [O3DE CLI](#using-the-o3de-command-line-tool-to-access-o3de-remote-repositories) to display,  download and register **Gems**, **Projects**, and **Templates** from **Remote Repositories**.
+
+## Using Project Manager to access O3DE remote repositories
+### Add a remote repository
+
+You can add existing O3DE remote repositories from the **Remote Sources** page in Project Manager which you can access from the **Engine** tab, or from the **Gem Catalog** right panel menu.
```
Comment (spham-amzn) [inline content/docs/user-guide/remote-content/use-a-remote-repository.md:14]:
```suggestion
You can add existing O3DE remote repositories using the **Remote Sources** page in Project Manager which you can access from the **Engine** tab, or from the **Gem Catalog** right panel menu.
```

File: content/docs/user-guide/remote-content/use-a-remote-repository.md [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+1. Press the **Add Repository** button. In the dialogue box that opens you can supply the URL or local path for a Gem repository.
```
Comment (spham-amzn) [inline content/docs/user-guide/remote-content/use-a-remote-repository.md:15]: Should use `en-us` terms (https://languagetool.org/insights/post/spelling-dialog-or-dialogue/)
 ```suggestion
1. Press the **Add Repository** button. In the dialog box that opens you can supply the URL or local path for a Gem repository. 
```

File: content/docs/user-guide/remote-content/use-a-remote-repository.md [hunk 8]
```diff
@@ -0,0 +1,163 @@
...
+### Activate or deactivate a remote repository
+
+You can toggle the visibility of remote content from a O3DE remote repository by pressing the visibility button on the same row as the O3DE remote repository on the **Remote Sources** page.  This is a useful way to hide content from that repository, without removing the remote repository.
+
+![Visibility buttons on the Remote Sources page](/images/user-guide/remote-content/ProjectManagerRemoteSources.JPG)
+
+### Remove a remote repository
+
+You can remove an O3DE remote repository URI from the **Remote Sources** page in the Project Manager so the remote content in it is no longer listed and the meta data is no longer updated.
+1. Select the O3DE remote repository you want to remove by clicking on the entry in the list of remote sources.
```
Comment (spham-amzn) [inline content/docs/user-guide/remote-content/use-a-remote-repository.md:34]: Add a blank line
```suggestion

1. Select the O3DE remote repository you want to remove by clicking on the entry in the list of remote sources.
```

File: content/docs/user-guide/remote-content/use-a-remote-repository.md [hunk 9]
```diff
@@ -0,0 +1,163 @@
...
+
+Gems available in remote repositories will also be downloaded automatically when activated in a project from the **Gem Catalog** page which can be accessed by pressing the **Configure Gems** button on the Edit Project Settings page for a project, or from the **Configure Gems...** project menu option on the **Projects** tab.
+
+### Download projects from a remote repository
+
+You can download projects from the **Projects** tab in the Project Manager by pressing the **Download** button on the project you want to download.  
+
+### Download templates from a remote repository
+
+You can download templates from the **Create a New Project** page in the Project Manager by selecting the project template in the list of project templates and pressing the  **Download Template** button in the right pane.
```
Comment (spham-amzn) [inline content/docs/user-guide/remote-content/use-a-remote-repository.md:51]:
```suggestion
You can download templates from the **Create a New Project** page in the Project Manager by selecting the project template in the list of project templates and pressing the **Download Template** button in the right pane.  
```

File: content/docs/user-guide/remote-content/use-a-remote-repository.md [hunk 10]
```diff
@@ -0,0 +1,163 @@
...
+
+### Download templates from a remote repository
+
+You can download templates from the **Create a New Project** page in the Project Manager by selecting the project template in the list of project templates and pressing the  **Download Template** button in the right pane.  
+
+
+## Using the `o3de` command line tool to access O3DE remote repositories 
+
+You can use the `o3de` command line tool to register, refresh, activate and deactivate remote repositories and download remote content.
+
```
Comment (spham-amzn) [inline content/docs/user-guide/remote-content/use-a-remote-repository.md:57]: Should we include an example for this like the other commands?

File: content/docs/user-guide/remote-content/use-a-remote-repository.md [hunk 11]
```diff
@@ -0,0 +1,163 @@
...
+Example which downloads the latest version of a project named `RemoteProject`:
+```cmd
+<engine>\scripts\o3de.bat download --project-name RemoteProject
+```
+
+Example which downloads the latest version of a template named `RemoteTemplate` into `c:/o3de-templates/RemoteTemplate`:
+```cmd
+<engine>\scripts\o3de.bat download --template-name RemoteTemplate  --dest-path c:/o3de-templates
+```
```
Comment (spham-amzn) [inline content/docs/user-guide/remote-content/use-a-remote-repository.md:133]: This looks like a windows example, use windows path separators instead
```suggestion
Example which downloads the latest version of a template named `RemoteTemplate` into `c:\o3de-templates\RemoteTemplate`:
```cmd
<engine>\scripts\o3de.bat download --template-name RemoteTemplate  --dest-path c:\o3de-templates
```
```

**Comment by spham-amzn** [comment]
Gems, Projects, and Templates is mentioned so often, maybe we should just call that feature 'GPT' to be more viral

File: content/docs/user-guide/remote-content/update-a-remote-repository.md (hunk 5 above)
Comment (AMZN-alexpete) [reply content/docs/user-guide/remote-content/update-a-remote-repository.md:91]: It is! But the "remote repository" is actually on your "local" computer, I'll rephrase it so it doesn't sound so wierd.

File: content/docs/user-guide/remote-content/use-a-remote-repository.md (hunk 10 above)
Comment (AMZN-alexpete) [reply content/docs/user-guide/remote-content/use-a-remote-repository.md:57]: The examples are in the sub sections, but I'll rephrase this and add a link to the official CLI docs

**Review by spham-amzn — APPROVED** [review]

**Pull request merged**